### DISCUSSION TITLE
Unable to start OKD upgrade, the clusterVersionOperator job pod in initcrashloopbackoff

### DISCUSSION THREAD

**Discussion by imdmahajankanika** [okd-project/okd#1683] [General]
**Describe the bug**

When tried to start OKD 4.11 upgrade, the version job created by clusterVersionOperator is creating a pod with securityContext annotation "openshift.io/scc: stackrox-collector" which is having "readOnlyRootFilesystem: true" and hence resulting in below error

mv: cannot remove '/manifests/0000_00_cluster-version-operator_xxxxxxxxxxx': Read-only file system

**Version**

4.11.0-0.okd-2022-07-29-154152, UPI Vsphere
**How reproducible**

Start upgrade to version 4.11.0-0.okd-2022-07-29-154152

**Log bundle**

**Comment by vrutkovs** [comment]
See https://access.redhat.com/solutions/5911951 and https://access.redhat.com/solutions/6985485

**Reply by imdmahajankanika** [reply]
@vrutkovs Thanks! For the workaround, I had put `readOnlyRootFilesystem: false` in SCC stackrox-collector, instead of deleting it.
